Models/models.js

```diff
@@ -36,4 +36,21 @@ exports.fetchAllArticles = () => {
       return rows
     })
 
-}
\ No newline at end of file
+}
+
+exports.fetchArticleCommentsById = (article_id) => {
+    const query = `SELECT comments.comment_id, 
+    comments.votes, comments.created_at, comments.author, comments.body, comments.article_id
+    FROM comments
+    LEFT JOIN articles ON articles.article_id= comments.article_id
+    WHERE comments.article_id = $1
```

Comment:
There is no need for a join here as all the information you need exists on the comments table 🙂